tests/frontier/opcodes/test_swap.py

```diff
@@ -0,0 +1,183 @@
+"""
+A State test for the set of `SWAP*` opcodes.
+Ported from: https://github.com/ethereum/tests/
+blob/develop/src/GeneralStateTestsFiller/VMTests/vmTests/swapFiller.yml.
+"""
+
+import pytest
+
+from ethereum_test_forks import Fork, Frontier, Homestead
+from ethereum_test_tools import (
+    Account,
+    Alloc,
+    Bytecode,
+    Environment,
+    StateTestFiller,
+    Transaction,
+)
+from ethereum_test_tools import Opcodes as Op
+
+
+class Storage:
+    def __init__(self):
+        self._data = {}  # Internal storage for key-value pairs
+
+    def store_next(
+        self, key: int, value: int, debug_hint: str | None = None
+    ) -> dict:
+        """
+        Store a value at the specified key in the storage.
+
+        Args:
+            key: The key where the value will be stored.
+            value: The value to store.
+            debug_hint: Optional debug hint to print when updating the storage.
+
+        Returns:
+            The updated storage.
+        """
+        self._data[key] = value  # Update the storage at the specified key
+
+        if debug_hint:
+            print(
+                f"Debug: {debug_hint} - Updated key {key} with value {value}"
+            )
+
+        return self._data  # Return the updated storage
+
+    def __getitem__(self, key):
+        """Allow accessing stored values using `storage[key]` syntax."""
+        return self._data[key]
+
+    def __setitem__(self, key, value):
+        """Allow updating stored values using `storage[key] = value` syntax."""
+        self._data[key] = value
```

Comment:
```suggestion
from ethereum_test_tools import Storage
```
No need to re-define this here, simply importing should suffice.